File: 009/Python/Problem009Tests.py
#!/usr/bin/env python

import unittest

import problem009

class Problem009Tests(unittest.TestCase):

    def testpythagorean_triplet(self):
        self.assertEquals(60, problem009.pythagorean_triplet(12))
        self.assertEquals(480, problem009.pythagorean_triplet(24))
        self.assertEquals(453960, problem009.pythagorean_triplet(234))

unittest.main()

File: 009/Erlang/problem009_tests.erl
-module(problem009_tests).

-include_lib("eunit/include/eunit.hrl").

pythagorean_triplet_test_() ->
  [
  ?_assertEqual(60, problem009:pythagorean_triplet(12)),
  ?_assertEqual(480, problem009:pythagorean_triplet(24)),
  ?_assertEqual(453960, problem009:pythagorean_triplet(234)),
  ?_assert(true)
  ].
